src/features/environmentDetails/components/Specification/SpecificationEdit.tsx

```diff
@@ -153,7 +170,10 @@ export const SpecificationEdit = ({
         ) : (
           <>
             <Box sx={{ marginBottom: "30px" }}>
-              <RequestedPackagesEdit packageList={requestedPackages} />
+              <RequestedPackagesEdit
+                packageList={requestedPackages}
+                onDefaultEnvIsChanged={onUpdateDefaultEnvironment}
```

Comment:
Does this update the packages list to reflect what is in the selected environment version?

Reply:
Yes, once you change the selected environment the Requested Packages will update